src/mono/wasm/debugger/BrowserDebugProxy/MonoSDBHelper.cs

```diff
@@ -764,6 +767,9 @@ internal sealed class MonoSDBHelper
 
         private readonly ILogger logger;
         private Regex regexForAsyncLocals = new Regex(@"\<([^)]*)\>", RegexOptions.Singleline);
+        private Regex regexForAsyncMethodName = new Regex(@"\<([^>]*)\>([d][_][_])([0-9]*)", RegexOptions.Singleline);
+        private Regex regexForGenericArgs = new Regex(@"[`][0-9]*");
+        private Regex regexForNestedLeftRightAngleBrackets = new Regex("^(((?'Open'<)[^<>]*)+((?'Close-Open'>)[^<>]*)+)*(?(Open)(?!))[^<>]*");
```

Comment:
These should be `static`.

Reply:
Maybe we should use use `RegexOptions.Compiled` here, since these are going to be used a lot.